gr-fec/lib/ccsds_encoder_impl.cc

```diff
@@ -28,9 +28,7 @@
 #include <gnuradio/fec/generic_encoder.h>
 #include <cstdio>
```

Comment:
as far as I can see, we should kill this `#include`, since it's not used within this file. Not your job (within this PR); but I certainly don't mind if you do.